addons/nvidia-bootstrap/image/run.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+
+# Simple early detection of nvidia card
+grep 10de102d /proc/bus/pci/devices || exit 0
```

Comment:
Never mind, the G2 support will be more involved -- that NVIDIA installer ignores the K520 GPU on G2s